lib/recipe/widgets/ingredient_form.dart

```diff
@@ -163,6 +199,7 @@ class IngredientMeasurements extends StatelessWidget {
               return Slidable(
                 key: Key(state.recipes[recipeIndex].ingredients[ingredientIndex]
                     .measurements[index].id),
+                enabled: !state.recipes[recipeIndex].saved,
```

Comment:
As with comments below, I'm thinking if we don't want to ditch slidable altogether in favour of long presses to be consistent across the whole app?